main.cpp

```diff
@@ -52,87 +51,96 @@ arm.rotateCounterClockwise();           // 4) demonstrates camelCase
 
 
 //  1)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun:  guitarist
+//  action 1:  the guitarist plays guitar
+guitarist.playGuitar();
+//  action 2:  the guitarist uses guitar padels
+guitarist.useGuitarPadels();
+//  action 3:  the guitarist wirtes songs
+guitarist.wirteSongs();
 
 //  2)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun:  cat
+//  action 1:  the cat eats fish
+cat.eatFish();
+//  action 2:  the cat runs
+cat.run();
+//  action 3:  the cat drinks water
+cat.drinkWater();
 
 //  3)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun:  egnineer
+//  action 1:  the engineer writes code
+engineer.writeCodes();
+//  action 2:  the engineer designs circuits
+engineer.designCircuit();
+//  action 3:  the engineer cooperates with team
+engineer.cooperateWithTeam();
 
 //  4)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun:  basketballPlayer
+//  action 1:  basketballPlayer plays games
+basketballPlayer.playGame();
+//  action 2:  basketballPlayer practices
+basketballPlayer.practice();
+//  action 3:  basketballPlayer watches films
+basketballPlayer.watchFilm();
 
 //  5)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:  manufacturer
+//  action 1:  manufacturer produces
+    manufacturer.produce();
+//  action 2:  manufacturer manufacures products
+    manufacturer.manufacureProduct();
+//  action 3:  manufacturer collaborates with brands
+    manufacturer.CollaborateWithBrand();
```

Comment:
function names and variable names should start with a lower-case letter.
`manufacturer.collaborateWithBrand();` is what you should write here.
- [ ] revise